### ISSUE TITLE
Demo: Menu highlights missing in iOS (take 2)

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#1589]
Same with new menu on mobile (old issue https://github.com/GSA/digitalgov.gov/issues/1077)

Android is still ok, but iOS still missing highlight when you tap:

![menu Image from iOS](https://user-images.githubusercontent.com/19961574/70069707-31b3ea00-15c0-11ea-860f-0043b707abb7.png)

--- 

Android: 

![menu Screenshot_20191203-112230_Chrome](https://user-images.githubusercontent.com/19961574/70069717-38426180-15c0-11ea-82ce-d35517539d55.jpg)
